<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_login">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="bienvenida">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bienvenido al chat Ciencias</string>
    </property>
   </widget>
   <widget class="QLabel" name="et_usser">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>52</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="et_ip">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>52</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>IP propia:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_usser">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_ip1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_ip2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="et_ip_2">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IP conectar:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
